src/mm-rating/mm-rating.js

```diff
@@ -0,0 +1,113 @@
+( function (scope) {
+    scope.Rating = Polymer({
+        is: 'mm-rating',
+        hovered: false,
+        properties: {
+            icon: {
+                type: String,
+                value: 'star'
+            },
+            customIcon: {
+                type: String,
+                value: null
+            },
+            iconWidth: {
+                type: Number,
+                value: 20
+            },
+            iconHeight: {
+                type: Number,
+                value: 20
+            },
+            rating: {
+                type: Number,
+                value: 0
+            },
+            displayTotals: {
+                type: Boolean,
+                value: false
+            },
+            displayGreetings: {
+                type: Boolean,
+                value: false
+            },
+            readonly: {
+                type: Boolean,
+                value: false
+            },
+            values: {
+                type: Object,
+                value: {0: 'Poor', 1: 'Need Improvement', 2: 'Reasonable', 3: 'Good', 4: 'Excellent'}
+            }
+        },
+
+        ready: function() {
+            this._defaultNoOfIcons = [];
+            if(typeof this.values === 'string'){
+                var greetObj = eval("("+ this.values +")");
```

Comment:
So the [Polymer.Base.deserialize](https://www.polymer-project.org/1.0/api/#Polymer.Base:method-deserialize) method should already do this assuming the input is valid JSON. We definitely don't want an eval anywhere in this code as its an entry point for XSS attacks.

I also feel like design wise we probably dont want to tightly couple the custom rollover labels with the 'length' of the rating.  Maybe allowing something like a `range` property where its a Number > 0 to specify the max star count, then allow the custom rollovers via Array.